ambry-store/src/main/java/com.github.ambry.store/StoreMessageReadSet.java

```diff
@@ -146,6 +148,27 @@ public void close() {
       segment.closeView();
     }
   }
+
+  /**
+   * Do data doPrefetch: from disk to memory buffer.
+   * @param relativeOffset the relativeOffset to start.
+   * @param size The size requested to doPrefetch.
+   */
```

Comment:
and that this throws?

Reply:
You mean comment?     @throws IOException?